fixtures/pages-functions-app/wrangler.toml

```diff
@@ -0,0 +1,48 @@
+# Generated by Wrangler on Mon Mar 25 2024 09:33:15 GMT+0000 (Greenwich Mean Time)
+name = "process-env"
+# pages_build_output_dir = "dist-test"
+compatibility_date = "2024-02-14"
+
+[env.production]
+compatibility_date = "2024-02-14"
+
+  [env.production.vars]
+  TEST_JSON = """
+{
+json: "value"
+}"""
+  TEST_PLAINTEXT = "PLAINTEXT"
+
+  [[env.production.kv_namespaces]]
+  id = "01962077d09242c5b0978a12bccaa5ed"
+  binding = "KV"
```

Comment:
Are these actual IDs from a CF account? Should we just use some mock values instead?

Reply:
I've removed this file